run.py

```diff
@@ -124,7 +124,7 @@
 # Other global variables
 ###############################################################################
 CORE_VERSION_URL = 'https://gae-init.appspot.com/_s/version/'
-INERNET_TEST_URL = 'http://74.125.228.100'
+INERNET_TEST_URL = 'http://google.com'
```

Comment:
as this is a redirect.. maybe it's better to use the https://www.google.com or it doesn't matter?